frontend/src/store/modules/files.js

```diff
@@ -153,7 +153,7 @@ const actions = {
     async createFolder({ getters, dispatch }, data) {
         let folderName = data;
         let client = getters.StateWebdavClient;
-        let path = getters.StatePath[getters.StatePath.length - 1].path + folderName;
+        let path = `${getters.StatePath[getters.StatePath.length - 1].path}/${folderName}`;
```

Comment:
@tailoric I have changed the implementation of this function. See my PR https://github.com/amosproj/amos2021ws03-teams-to-nextcloud/pull/62 . I think you can close this PR.